StripeFinancialConnections/StripeFinancialConnections/Source/Web/FinancialConnectionsWebFlowViewController.swift

```diff
@@ -165,7 +165,9 @@ extension FinancialConnectionsWebFlowViewController {
                         {
                             let instantDebitsLinkedBank = InstantDebitsLinkedBankImplementation(
                                 paymentMethodId: paymentMethodId,
-                                bankName: Self.extractValue(from: returnUrl, key: "bank_name"),
+                                bankName: Self.extractValue(from: returnUrl, key: "bank_name")?
+                                // backend can return "+" instead of a more-common encoding of "%20" for spaces
+                                    .replacingOccurrences(of: "+", with: " "),
```

Comment:
this is a small change that mitigates the impact of `.replacingOccurrences(of: "+", with: " ")` to only the `bank_name`; just defensive programming

More context is here: https://github.com/stripe/stripe-ios/pull/3618